00new_packages.autobuild

```diff
@@ -1,4 +1,7 @@
 # IMPORTANT: new packages must be added at the top of this file
 # They must NEVER be added to other files
 
-
+cmake_package 'gui/rock_webapp'
+ruby_package 'tools/rest_api' do |pkg|
+    Autoproj.env_add_path 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
```

Comment:
OK then. If it's not a path, I'm all OK with the current envvar name. Only, don't use env_add_path but env_set. Migration to a hypothetical ROCK_PREFIX_PATH would obviously require too much time to be relevant here anyways. Was just a point for future discussions.